### ISSUE TITLE
Some twisted.python.logger submodules are not installed on Python 3, breaking Twisted on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7563]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#7563|
|Type|defect|
|Created|2014-08-21 21:38:29Z|
|Branch|https://github.com/twisted/twisted/tree/log-py3-7563|

E.g. `twisted.python.logger._tzhelper` is not installed.

<details><summary>Searchable metadata</summary>

```
trac-id__7563 7563
type__defect defect
reporter__itamarst itamarst
priority__normal normal
milestone__New_Logging_System New Logging System
branch__log_py3_7563 log-py3-7563
branch_author__ 
status__closed closed
resolution__fixed fixed
component__logger logger
keywords__ 
time__1408657109635491 1408657109635491
changetime__1481165779520945 1481165779520945
version__None None
owner__wsanchez wsanchez

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @glyph|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @wsanchez|
|-|-|

Umm not a regression any more, oops.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_5"></a>|@wsanchez commented|
|-|-|

Is it safe to assume that if the Py3 builder is green, that this bug isn't a problem?

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_6"></a>|@glyph commented|
|-|-|

Unfortunately, no.  The py3 builder does not test our installation procedure.  This is a bug in the build infrastructure, but it's a tricky one to fix without a big overhaul (that nobody seems to want to do, myself included).  I think this big overhaul is coming soon, but the problem is basically this: twisted works, the tests pass, but the tests run against a source checkout and not against a built artifact.  The bug here is that the built artifact is missing some files because the relevant declarations were not made.  You can test locally by doing a "`mktmpenv; pip install .../twisted-checkout`" (no `-e` option, that's important) and then attempting to import stuff.  Normally at that point you could just run `trial`, but of course... no `trial` on py3 yet.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_9"></a>|@wsanchez commented|
|-|-|

r44396 adds the missing modules.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez removed owner<br>@wsanchez set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_11"></a>|@ralphm set owner to @wsanchez|
|-|-|

Yay. Please merge with the logger branch.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_12"></a>|@wsanchez set status to `closed`|
|-|-|

(In [44416]) Merge branches/log-py3-7563: Missing logger modules in dist3.py

Author: wsanchez
Reviewer: ralphm
Fixes: [#7563](https://github.com/twisted/twisted/issues/7563)
